candies/cli/args/flag.py

```diff
@@ -28,7 +28,7 @@ class Flag:
         def git():
             ...
 
-        @git.command
+        @cli.command
```

Comment:
So, how the subcommands are defined now?

For example, what kind of syntax does the `dotnet tool install` represent?
```py
@cli
def dotnet():
    ...

@cli.command
def tool():
    ...

@tool.command # (?)
def install():
    ...
```

If so, doesn't it look a bit confusing?

Reply:
The syntax for defining subcommands has not changed.

Reply:
I think we should always prefer the "old" style, because it's more consistent with hierarchical nature of commands. Thus, it'd probably be better to use it in comments as well. What do you think?

Reply:
I think that this will make the API more consistent. Will fix this, thank you!